bundler/lib/dependabot/bundler/update_checker.rb

```diff
@@ -75,7 +75,7 @@ def updated_requirements
 
       def requirements_unlocked_or_can_be?
         return true if requirements_unlocked?
-        return false if requirements_update_strategy == :lockfile_only
+        return false if requirements_update_strategy == "lockfile_only"
```

Comment:
I know this already exists and you are helping out; could you please consider making these constants? That'll minimize the possibility of this regressing and help us standardize on known update strategies too (one place to look for them and change them)

Reply:
I added the `RequirementsUpdateStrategy` enum and used it everywhere instead of strings or symbols.